### PR TITLE
[Android and iOS] Update Mobile documentation

### PR DESCRIPTION
**Pull request by darcywong00**
* Update the Android and iOS readme.md files to mention the KMW build pre-requisites.
* Update the Android in-app info for 10.0 content (copied from https://help.keyman.com/products/android/10.0/)

### CODE REVIEW

File: ios/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * iOS 8+
 * SwiftLint (`brew install swiftlint`)
 * Carthage (`brew install carthage`)
+* [Node.js](https://nodejs.org/) 8.9+ (for building KeymanWeb)
```
Comment (mcdurdin) [inline ios/README.md]: Worth referencing the KMW requirements in the same way as with the android README.md

File: ios/README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ The source for our published iOS Keyman app is found within the **keyman/** subd
 the views specific to the Keyman app, while the core functionality is found in KeymanEngine.framework.
 
 ### Building
+Building Keyman Web is a precursor for compiling KMEI, so verify your system has all the [Minimum Web Compilation Requirements](../web/README.md#minimum-web-compilation-requirements)
```
Comment (mcdurdin) [inline ios/README.md]: We need this line in the android README.md too, right?

Comment (darcywong00) [reply ios/README.md]: yep, I've already added it in the section

## Keyman for Android Development

Comment (mcdurdin) [reply ios/README.md]: Sorry, my mistake ...

**Pull request merged**